internal/deploy/yaml_test.go

```diff
@@ -0,0 +1,281 @@
+package deploy
+
+import (
+	"os"
+	"testing"
+
+	"github.com/shipa-corp/ketch/internal/testutils"
+
+	ketchv1 "github.com/shipa-corp/ketch/internal/api/v1beta1"
```

Comment:
is it possible to group imports? 
standard + third-party + current project

Reply:
Fixed. Will have to look at my IDE's config.